<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindowc</class>
 <widget class="QMainWindow" name="mainwindowc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GÃ©rer les commandes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/resources/img/image2.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>891</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter commande</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>591</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Ajouter une commande</string>
      </property>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>90</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_descri">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>130</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>description</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>260</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_dispo">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>170</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>dispo</string>
       </property>
      </widget>
      <widget class="QLabel" name="image">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>150</y>
         <width>381</width>
         <height>161</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Afficher commande</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>701</width>
        <height>391</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Afficher une commande</string>
      </property>
      <widget class="QTableView" name="tab_commande">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>581</width>
         <height>211</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pdf">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>290</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>350</y>
         <width>83</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>trie</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>250</y>
         <width>271</width>
         <height>80</height>
        </rect>
       </property>
       <property name="title">
        <string>rechercher</string>
       </property>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>rechercher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="rech">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QComboBox" name="valeur">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>20</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>nom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>id</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>290</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>DISPONABILITY</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>290</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Export Excel</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Mdifier/supprimer commande</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>801</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Modifier une commande</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_id_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>230</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>description</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nom_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>160</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_descri_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>230</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_update">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>390</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Modifier commande</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>50</y>
         <width>391</width>
         <height>451</height>
        </rect>
       </property>
       <property name="title">
        <string>Supprimer une commande</string>
       </property>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>121</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>id</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_id_supp">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>110</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_supprimer">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>190</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Supprimer commande</string>
        </property>
       </widget>
      </widget>
      <widget class="QLineEdit" name="lineEdit_dispo_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>280</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>280</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>dispo</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>livraison</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>701</width>
        <height>561</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>ajouter une commande (par id)</string>
      </property>
      <widget class="QLineEdit" name="le_id_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QTableView" name="tab_commande_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>561</width>
         <height>141</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>200</y>
         <width>661</width>
         <height>371</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="title">
        <string>Modifier une commande</string>
       </property>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>id</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>70</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>230</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>description</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom_3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>160</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_descri_3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>230</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_update_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>310</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Modifier commande</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>50</y>
          <width>391</width>
          <height>451</height>
         </rect>
        </property>
        <property name="title">
         <string>Supprimer une commande</string>
        </property>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>121</width>
           <height>71</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>id</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_id_supp_2">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>110</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_supprimer_2">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>190</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Supprimer commande</string>
         </property>
        </widget>
       </widget>
       <widget class="QLineEdit" name="lineEdit_dispo_3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>280</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>280</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>dispo</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_pp">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>40</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>912</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
